cni-plugin/integration/run.sh

```diff
@@ -29,6 +29,12 @@ function cleanup() {
     k delete ns cni-plugin-test || echo "could not delete namespace cni-plugin-test"
 }
 
+function install_calico() {
+  echo '# Installing Calico...'
+  local yaml="https://k3d.io/v5.3.0/usage/advanced/calico.yaml"
+  k apply -f $yaml || echo "could not apply $yaml"
```

Comment:
for cleanup, you can add a function `cleanup_calico` and then call `trap cleanup_calico EXIT` here

Reply:
Nice, I ended up doing it all in the same cleanup, perhaps a separate one might make more sense though... lmk what you think.